src/pages/components/Label.tsx

```diff
@@ -2,7 +2,11 @@ import React from 'react'
 import styled from 'styled-components'
 
 const StyledDiv = styled.div`
-  min-width: 80px;
+  display: inline-flex;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
```

Comment:
The labels are centered, better to align them to right by "justify-content: flex-start;"
before
![image](https://github.com/hasadna/open-bus-map-search/assets/76536506/b8f17421-ad51-4558-92a9-665f5230d038)
after
![image](https://github.com/hasadna/open-bus-map-search/assets/76536506/c444ffae-ce6b-42a4-9075-9ec2bd726361)

Reply:
done. thanks.